26.md

```diff
@@ -4,25 +4,61 @@ NIP-26
 Relays List
 -----------
 
-`draft` `optional` `author:fiatjaf`
+`draft` `optional` `author:fiatjaf` `author:cameri`
 
 A special event with kind `10001`, meaning "relay list" is defined as having a list of tags, one for each relay the author uses.
 
 The content is not used.
 
-The tags consist of arrays of 3 elements: the first is the relay URL, the second is the "read" condition -- which if set `"true"` means the client is reading events from that relay --, the third is the "write" condition -- which if set to `"true"` means the user is writing events to that relay.
+The tags consist of arrays of 3 elements: the first is the relay URL, the second is the "read" condition, the third is the "write" condition.
 
-In the future this NIP will be modified with more fine-grained conditions for writing and reading from relays, based on properties of the filter or of the event respectively, written as strings in a simple language.
+The "read" condition consists of a string containing a rule that follows a subset of the [runes](https://pypi.org/project/runes/) language. Specifically only the `|`, `&`, `!`, `>`, `<`, and `=` operators are allowed. When the rule is an empty string it evaluates to `true`. When it consists of a single naked `"!"` it evaluates to `false`. The `#` specifically must not be interpreted as a comment, but always as a tag attribute. All the operatores must be tested against all possible values in the case of filter values that consist of lists of values and also in the case of event tags that can have multiple values for the same tag -- in other words, the `=` may be interpreted as a `values.any(v => v == runeValue)` instead of an `values == runeValue` operator; the `<` may be interpreted as a `values.any(v => v < runeValue)` and so on. In the case of events tags must also be identified by the `#` prefix, just like in filters.
```

Comment:
Let's please include `^` aka begins with. With it it's possible to use prefixes for pubkeys and event IDs.

Reply:
Would it be easier if we drop the "!" evaluating to false and just pass some condition that will always evaluate to false, like `kind<0`?

Reply:
Ah, `\` is not allowed in FIELDNAME. I thought I was getting somewhere!

Reply:
Any fieldname other than `kind`, `pubkey`, `created_at`, `id` could mean you are searching in tags so there's no need to include the `#` to prefix them.